src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int COUNT_OF_RANDOM_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 1; i < COUNT_OF_RANDOM_BALLS + 1; i++) {
```

Comment:
This is overcomplicating things
```suggestion
        for (int i = 0; i < COUNT_OF_RANDOM_BALLS; i++) {
```